charts/nri-bundle/README.md

```diff
@@ -1,6 +1,6 @@
 # nri-bundle
 
-![Version: 4.6.0](https://img.shields.io/badge/Version-4.6.0-informational?style=flat-square)
```

Comment:
can you remove this line from the template and the readme? So that we do not have to manually update that